compiler/rustc_span/src/def_id/stable_crate_id_map.rs

```diff
@@ -0,0 +1,8 @@
+use std::hash::BuildHasherDefault;
+
+use rustc_data_structures::unhash::Unhasher;
+
+use super::{CrateNum, StableCrateId};
+
+pub type StableCrateIdMap =
+    indexmap::IndexMap<StableCrateId, CrateNum, BuildHasherDefault<Unhasher>>;
```

Comment:
This is probably not worth a file any more.
Could you put the type alias directly in `rustc_span::def_id`?

Reply:
done